config.toml

```diff
@@ -25,6 +25,15 @@ contentDir = "/content/"
 docsDir = "/docs/"
 sourceRepo = "https://github.com/o3de/o3de"
 
+[params.docs]
+# Docs version data format: ["display name", "baseURL"]
+# Note: A version that has "www.o3de.org" in its baseURL will get " (latest)" appended to its display name in the version selector.
+versions = [
+  ["22.10", "https://www.o3de.org"],
+  ["development", "https://development--o3deorg.netlify.app"],
+  ["22.05", "https://2205--o3deorg.netlify.app"]
```

Comment:
Is there any way to get subdomains for these? Like `development.o3de.org` and `2205.o3de.org`

Reply:
There is! We have to have the Linux Foundation set up the DNS records. So far we do not have a contact to do so, even after months of trying.

Reply:
It would probably be best to just go live with the final subdomains, but I understand that it could mean waiting for a long time on this... so if the consensus is to use the temp ones in the meantime, I'm fine with it.

Question, does this change actually need to be applied on those versions of the site as well? Or do they share the same sass/js?

Reply:
This change will need to be merged / applied to every branch that is selectable in the dropdown (i.e. main, development, and 2205) - and eventually stabilization when that's created for the next release.